part1.kt

```diff
@@ -0,0 +1,14 @@
+
+fun main(args: Array<String>) {
+   fun getLength(str: String?) : Int {
+    if(str != null)

```

Comment:
There is a problem here, try to correct it

Reply:
fun main(args: Array<String>) {
    fun getLength(str: String?) : Int {
        if(str != null)

            return (str. length)
        else
            return 0
    }

    fun main(args: Array<String>) {
        println(getLength("This is a string!"))
    }
}